tests/config/un.test.yaml

```diff
@@ -39,7 +39,7 @@
     - pool: weight
   ttl: 300
   type: CNAME
-  value: cl-dabdb3fc.edgecdn.ru.
+  value: cl-dabdb1fc.edgecdn.ru.
```

Comment:
What's this change for? I don't see any corresponding changes in the tests below.